packages/cardhost/app/templates/components/cards/cardstack/base-card/embedded.hbs

```diff
@@ -1,9 +1,12 @@
 {{!-- TODO move this into the @cardstack/base-card package when we support runtime js compilation --}}
-
-{{!-- TODO use a sanitized version of the @card.id in the class name after we have css coming from card documents --}}
 <div
   data-test-embedded-card={{@card.name}}
-  class="embedded-card cardstack_base-card {{@mode}}"
+  class="embedded-card
+    embedded-card--{{@mode}}
+    cardstack_base-card-embedded
+    {{safe-css-string @card.name}}-embedded
+    {{safe-css-string @card.adoptedFromName}}-embedded
+  "
```

Comment:
The trailing double quote on its own line here and in isolated.hbs seems to cause some strange results in the themer:

<img width="814" alt="Screen Shot 2019-12-31 at 2 16 13 PM" src="https://user-images.githubusercontent.com/16627268/71631709-214f5900-2bd8-11ea-9638-d370f6db326a.png">